### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by GarvG**
I Made avatar and alert! Have a look, Suggestions appericated!

### CODE REVIEW

File: alert.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+.content-background
+{
+    height: fit-content;
+    width: 900px;
+    border-radius: 15px;
+    box-shadow:5px 10px 8px #888888;
+    padding: 20px;
+    margin-bottom: 10px;
+}
+
```
Comment (amartya-rs) [inline alert.css:20]: remove unnecessary spacing

File: avatar.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+{
+    text-align: center;
+}
+main{
+    text-align: center;
+   
+    
+}
+
+.XXL
```
Comment (amartya-rs) [inline avatar.css:13]: Be consistent with the class names I could see both small and capital letters. I suggest you to use small case for naming.

File: alert.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    border-radius: 15px;
+    box-shadow:5px 10px 8px #888888;
+    padding: 20px;
+    margin-bottom: 10px;
+}
+   
+
+
+
+.content.content-background.D
```
Comment (amartya-rs) [inline alert.css:24]: Remove unnecessary spacing.

File: avatar.css (hunk 2 above)
Comment (amartya-rs) [inline avatar.css:13]: Be consistent with the class names I could spot both small and upper case. I would suggest you to use small case for naming.

File: avatar.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+                <div class="center-container">
+                    <h3 class="headers">Round Avatar!</h3>
+                    <div>
+                        <span> <img class="XXL" src="garv.png"></span>
+
+                        <span> <img class="XL" src="garv.png"></span>
+                        <span> <img class="L" src="garv.png"></span>
+                        <span> <img class="M" src="garv.png"></span>
+                        <span> <img class="S" src="garv.png"></span>
```
Comment (rutu14) [inline avatar.html:51]: Try using the same image as square ones and change the look through border radius

File: alert.css (hunk 3 above)
Comment (GarvG) [reply alert.css]: Done

**Review by sudipkundu999 — APPROVED** [review]
Just one suggestion:
In the deploy preview, 
the square avatars look a little distorted 
as they are not maintaining the original aspect ratio. 
You can use `object-fit` to solve this.
Refer this: https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

File: avatar.css (hunk 2 above)
Comment (GarvG) [reply avatar.css:12]: Sure ,Next Time I'll keep note on that!

File: avatar.html (hunk 4 above)
Comment (GarvG) [reply avatar.html]: I changed and used same image both for my round and square avatar

**Comment by sudipkundu999** [comment]
for garv
This is what I meant Garv, 
the original aspect ratio is 3:4 
but the rendered aspect ratio is 1:1 
means your picture is getting squeezed to fit, 
if you want that then it's okay but,
if you want to maintain the actual then `object-fit` can be used.

**Comment by GarvG** [comment]
> if you want that then it's okay but,
> if you want to maintain the actual then `object-fit` can be use

File: alert.css [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    font-size: 1.5rem;
+    font-weight: 700;
+    text-align: center;
+}
+.content-background
+{
+    height: fit-content;
+    width: 900px;
+    border-radius: 15px;
+    box-shadow:5px 10px 8px #888888;
```
Comment (iRohitGaur) [inline alert.css]: use variable if you are re-using this color someplace else aswell

File: alert.css [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    margin-bottom: 10px;
+}
+   .content.content-background.D
+{
+    background-color: var(--danger);
+}
+.content.content-background.S
+{
+   background-color: var(--success);
+
```
Comment (iRohitGaur) [inline alert.css:25]: remove extra spacing and get consistency in the bracket structure

File: alert.css (hunk 5 above)
Comment (GarvG) [reply alert.css]: done with changes

File: alert.css (hunk 6 above)
Comment (GarvG) [reply alert.css:25]: done removing extra spaces

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

**Pull request merged**